ChecklistItem[] checklist
---
bool success
string message